plugins/org.polarsys.capella.scenario.editor.doc/html/TextualEditor/Functionalities.mediawiki

```diff
@@ -56,6 +58,7 @@ The user can create messages between participants assuming the fact that the exc
 ::: "A 1" -> "A 2" ''' withExecution ''' : "exec"
 ::: "A 2" -> "A 3"
 ::: '''deactivate''' "A 2"
+* ''' return branch ''': to mark that a message has a return,the '''withReturn''' keyword can be used; if the message does not return immediately and other events ocur on its execution, then the '''withExecution''' keyword can be used in the same time with the '''withReturn''' keyword to mark that the current message is a complex message and has return.
```

Comment:
space after "," : has a return, the ....

Reply:
???

Reply:
has a return instead of has return?

Reply:
ok